### ISSUE TITLE
WCAG 1.4.3: Contrast (Minimum)

### ISSUE THREAD

**Issue by arnfinn** [mong/mongts#1085]
Observations:

Contrast with minimum ratio failed

"Variabler", "Filter" and "Andre innstillinger" https://www.skde.no/pasientstrommer/

https://www.skde.no/kvalitetsregistre/alle/sykehus
